resources/assets/js/app/views/votes/CreateVote.js

```diff
@@ -1,5 +1,142 @@
+var Backbone = require('backbone');
 var Marionette = require('backbone.marionette');
+var Radio = require('backbone.radio');
+var moment = require('moment');
+
+var DateHelper = require('../../helpers/dateHelper.js');
+
+var currentUser = require('../../initializers/currentUser');
+
+var CreateVoteItemCollection = require('./CreateVoteItemCollection');
+var userCollectionView = require('../users/userCollection');
+
 
 module.exports = Marionette.LayoutView.extend({
-    template: 'voteCreateLayout'
+    className: 'well',
+    template: 'voteCreateLayout',
+    regions: {
+        answers: '#vote-answers',
+        voteAcessedUsers: '#vote-access-users',
+        voteNotAccessedUsers: '#vote-new-addUsers'
+    },
+    ui: {
+        add: '#addAnswer',
+        start: '#start',
+        title: '#question-title',
+        errors: '.js-errors',
+        tags: '#tags',
+        isPublic: 'input[name=access]:checked',
```

Comment:
in this row in isPublic you will have checkbox/radio only if it's checked, you should check this in code